explainer.md

```diff
@@ -4,11 +4,11 @@
 
 The Highlight API extends the concept of [CSS Highlight Pseudo-elements](https://drafts.csswg.org/css-pseudo-4/#highlight-pseudos) by providing a way for web developers to style the text of arbitrary Range objects, rather than being limited to the user-agent defined ```::selection```, ```::inactive-selection```, ```::spelling-error```, and ```::grammar-error```. This is useful in a variety of scenarios, including editing frameworks that wish to implement their own selection, find-on-page over virtualized documents, multiple selection to represent online collaboration, and spellchecking frameworks.
 
-Current browsers do not provide this functionality which forces web developers and framework authors to modify the underlying structure of the DOM tree in order to achieve the rendering they desire. This can quickly become complicated in cases where the desired highlight/selection spans elements across multiple subtrees, and requires DOM updates to the view in order to adjust highlights as they change. The Highlight API provides a programmatic way of adding and removing highlights that do not affect the underlying DOM structure, but instead applies styles based on Range objects.
+Current browsers do not provide this functionality which forces web developers and framework authors to modify the underlying structure of the DOM tree in order to achieve the rendering they desire. This can quickly become complicated in cases where the desired highlight/selection spans elements across multiple subtrees, and requires DOM updates to the view in order to adjust highlights as they change. The Highlight API provides a programmatic way of adding and removing highlights that do not affect the underlying DOM structure, but instead applies styles based on HighlightRangeGroup objects (introduced by this proposal), which contain Range objects.
 
 ## Example usage
 
-The following code uses the ```::highlight``` pseudo-element to apply a yellow background and blue foreground color to the text ```One two```. It does so by adding a Range into the **HighlightsMap** (a new concept introduced by this proposal).
+The following code uses the ```::highlight``` pseudo-element to apply a yellow background and blue foreground color to the text ```One two```. It does so by adding a Range into a HighlightRangeGroup, which is then inserted into the **HighlightsMap** (a new concept introduced by this proposal).
```

Comment:
Use `**HighlightRangeGroup**` for consistency.

Let's rearrange the wording a bit

"It does so by adding a **HighlightRangeGroup** to the **HighlightsMap** (both of these are new concepts introduced by this proposal). The HighlightRangeGroup will contain a Range whose boundary points surround the text ```One two```.